docs/src/dev/io/graphson.asciidoc

```diff
@@ -19,9 +19,11 @@ limitations under the License.
 
 image:gremlin-graphson.png[width=350,float=left] GraphSON is a JSON-based format that is designed for human readable
 output that is easily supported in any programming language through the wide-array of JSON parsing libraries that
-exist on virtually all platforms. GraphSON is considered both a "graph" format and a generalized object serialization
-format. That characteristic makes it useful as a serialization format for Gremlin Server where arbitrary objects
-of varying types may be returned as results.
+exist on virtually all platforms. GraphSON versions 1 to 3 were considered to be both a "graph" format and a
+generalized object serialization format. That characteristic makes it useful as a serialization format for Gremlin
+Server where arbitrary objects of varying types may be returned as results. However, starting in GraphSON 4, GraphSON
+is only intended to be a network serialization format that is only able to serialize specific types defined by the
+format. It is only meant to be used between language variants and the Gremlin Server.
```

Comment:
> It is only meant to be used between language variants and the Gremlin Server.

is it just language variants? that seems to rule out pure http based requests, no?

Reply:
Is it fair to say that GraphSON 4 is a useful "graph format" for any graph which is limited to containing TinkerPop serializable types?

Reply:
> is it just language variants? that seems to rule out pure http based requests, no?
updated to say that its intended to work with the HTTP API

> Is it fair to say that GraphSON 4 is a useful "graph format" for any graph which is limited to containing TinkerPop serializable types?
sure, but I don't think it would ever be used that way so I wouldn't phrase it that way